py/README.md

```diff
@@ -0,0 +1,76 @@
+# Deephaven Python Development
+
+## Requirements
+
+- Python >=3.8
+
+## Setting up Virtual Environment
+
+For any of the options below, you should set up a virtual environment in Python first. Once created, you must activate the virtual environment in each shell you open each time.
+
+```sh
+python3 -m venv .venv # Create. Only needed once
+source .venv/bin/activate # Activate the virtual environment
+deactivate # Exit virtual environment when done
+```
+
+**Note:**
+
+- This is not an exhaustive guide to managing python environments
+- Depending on your OS and how your PATH is setup, you may need to use `python3`, or a path to the explicit python version you want to use
+- You may choose to setup a "permanent" virtual environment location
+- You'll need to re-install the wheel anytime you are making python code changes that affect the wheel
+- `pip` can be a pain if you are trying to (re-)install a wheel with the same version number as before
+  - A `pip install --force-reinstall --no-deps "py/server/build/wheel/deephaven_core-<version>-py3-none-any.whl[autocomplete]"` may do the trick
+- You can install other python packages in your venv using `pip install <some-other-package>`
+- You can setup multiple virtual environments, and switch between them as necessary using `source /path/to/other-venv/bin/activate`
+- You can de-activate the virtual environment by running `deactivate`
+- You can use the `VIRTUAL_ENV` environment variable instead of sourcing / activating virtual environments: `VIRTUAL_ENV=/my/venv ./gradlew server-jetty-app:run`
```

Comment:
```suggestion
- You can use the `VIRTUAL_ENV` environment variable instead of sourcing/activating virtual environments: `VIRTUAL_ENV=/my/venv ./gradlew server-jetty-app:run`.
```